code/processes/wdb.q

```diff
@@ -240,7 +240,10 @@ handler:{
 /- notify the gateway that we are done
 flushend:{
     if[not @[value;`.wdb.reloadcomplete;0b];
-     @[{neg[x]"";neg[x][]};;()] each key reloadsummary;
+     $[eodwaittime>0;
+      @[{neg[x]"";neg[x][]};;()] each key reloadsummary;
+      @[{x"";x[]};;()] each key reloadsummary
```

Comment:
is this needed? if we sent sync requests, we shouldn't have to flush the handles?

Reply:
Hadn't thought of that. Apologies.